backend/alembic/versions/2023_02_01_2146-9e7ec4a9e3f2_add_skip_bool_skip_reason_to_task.py

```diff
@@ -0,0 +1,30 @@
+"""add skip bool & skip_reason to task
+
+Revision ID: 9e7ec4a9e3f2
+Revises: 7b8f0011e0b0
+Create Date: 2023-02-01 21:46:49.971052
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9e7ec4a9e3f2"
+down_revision = "7b8f0011e0b0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("task", sa.Column("skipped", sa.Boolean(), server_default=sa.text("false"), nullable=False))
+    op.add_column("task", sa.Column("skip_reason", sqlmodel.sql.sqltypes.AutoString(length=515), nullable=True))
```

Comment:
why is this 515 long?

Reply:
haha .. it should have been 512 .. I will change that.